include/verilated_funcs.h

```diff
@@ -495,10 +501,8 @@ static inline void VL_ASSIGNBIT_WO(int bit, WDataOutP owp) VL_MT_SAFE {
 #define VL_ASSIGN_SBW(obits, svar, rwp) \
     { \
         sc_biguint<(obits)> _butemp; \
-        for (int i = 0; i < VL_WORDS_I(obits); ++i) { \
-            int msb = ((i + 1) * VL_IDATASIZE) - 1; \
-            msb = (msb >= (obits)) ? ((obits)-1) : msb; \
-            _butemp.range(msb, i* VL_IDATASIZE) = (rwp)[i]; \
+        for (int i = 0; i< (obits); ++i) { \
+            _butemp.set(i, (rwp).test(i)); \
```

Comment:
Here and above it is now bit-by-bit which is 32x slower than word by word.

Is the raw data structure available?  If so you can call _vl_insert_WW pointing at the raw byte destination array.  If not perhaps need to copy that code - if so we will need to have tests to make sure each of the branches in that code are covered.

Reply:
I agree there is a slowdown, but it is safe. 
I tried using `sc_unsigned::get_raw()` , but for `sc_biguint` it didn't behave as expected. 
E.g.

```
sc_uint<32> i{0x55667788};
sc_biguint<128> s{0x00};
s = i*i;
  
cout << "0x" << hex << s << endl;
auto* ptr = s.get_raw(); // type is uint*
  
cout << "0x" <<*ptr << endl;
ptr++;
  
cout << "0x" << *ptr << endl;
```

and the output I get is 
```
0x000000000000000001c7d3463c82fb840 // correct
0x82fb840 // which is 3.5 bytes?
0x31f4d18f // which is something I have no idea what
```
Did I do something wrong?
Do you have some other hint?

Reply:
Looking at the systemC sources they use 30 BITS_PER_DIGIT not 32.  I didn't poke around to see what the extra 2 bits are being used for.

Anyhow I suggest you have something like

    while (lsb < size - BITS_PER_DIGIT)
         // Process BITS_PER_DIGIT chunk (30 bits)
         const uint32_t = get_datap(chunk++);
         _vl_insert_wi(out, data, lsb+BITS_PER_DIGIT-1, lsb)
         lsb += BITS_PER_DIGIT
    if (lsb) then read final chunk and mask off the appropriate number of output bits